src/app/page.module.css

```diff
@@ -15,7 +15,7 @@
   max-width: var(--max-width);
   width: 100%;
   z-index: 2;
-  font-family: var(--font-mono);
+  /* font-family: var(--font-mono); */
```

Comment:
Why is it disabled?

Reply:
This file was created previously but is no longer used. It might be better to just delete it.

Reply:
Yes, please delete it